tools/tasks/seed/lang.merge.ts

```diff
@@ -0,0 +1,63 @@
+import * as gulp from 'gulp';
+import * as gulpLoadPlugins from 'gulp-load-plugins';
+import * as runSequence from 'run-sequence';
+import { join } from 'path';
+import Config from '../../config';
+
+const plugins = <any>gulpLoadPlugins();
+const sourceElements: any[] = [];
+let langEmpty: boolean = true;
+
+gulp.task('get-source', function() {
```

Comment:
Dividing it into three different files doesn't seem like a bad idea. It'll follow the current structure and I don't think it'll be an overhead.

In the same time only a single task is being exported, so I suppose it's better to leave it the way it is.

Would you change the task names to have some special prefix to make sure we won't override custom user tasks? A suffix indicating that this is an internal task regarding the i18n seems enough (`_i18n`, `internal.i18n`, `seed.i18n`, whatever).